styleguide/javascript/README.md

```diff
@@ -82,22 +88,123 @@ Use arrows for functions that are passed as callbacks, since arrow functions bin
 
 If you're using function-style components in React, you (usually) don't need `this`, so it's better to use named functions because they are easier to read. But always prefer arrows instead of `.bind(this)`, and for unnamed closures.
 
+### camelCase in object properties
+
+Do:
+
+``` javascript
+const obj = {
+  fooBar: {
+    barBaz: 123
+  },
+}
+```
+
+Don't:
+
+``` javascript
+const obj = {
+  foo_bar: {
+    bar_baz: 123
+  },
+}
+```
+
+If you're working with an API that uses snake\_case, there are libraries that can transform keys between camelCase/snake\_case in nested objects. Just plug them into your API client.
+
 ## React
 
-TODO: Merge this with [our React styleguide](/styleguide/react). That guide may be outdated and uses CoffeeScript.
+*TODO: Merge this with [our React styleguide](/styleguide/react). That guide may be outdated and uses CoffeeScript.*
+
+Read and understand [the React documentation](https://reactjs.org/docs/hello-world.html). It's pretty short.
 
 ### Wrap your app in `<React.StrictMode>`
 
 This will enable some extra warnings in development mode.
 
 [React.js docs about Strict Mode](https://reactjs.org/docs/strict-mode.html)
 
+### Prefer function components over class components
```

Comment:
No opinion on this – happy for this part to be merged without the CSS stuff (for now).